src/components/crossDevice/GenericError/index.js

```diff
@@ -0,0 +1,22 @@
+import { h, Component } from 'preact'
+
+import theme from '../../Theme/style.css'
+import style from './style.css'
+import { sendScreen } from '../../../Tracker'
+
+class GenericError extends Component {
+  componentDidMount() {
+    sendScreen(['generic_client_error'])
+  }
+  render () {
+    return (
+      <div className={style.base}>
```

Comment:
I think this should use `theme.step` instead

Reply:
True!